.github/workflows/build-and-test.yaml

```diff
@@ -22,9 +31,19 @@ jobs:
           snapshots_repo="${repo_base}/snapshots"
           releases_repo="${repo_base}/releases"
 
+          if ${{ inputs.release || false }}; then
```

Comment:
What is the purpose of `|| false`? Won't it anyways always depend on `inputs.release` evaluating to `true`? Also, is this testing artefact intended to be still part of the PR?

Reply:
if `release` is not defined (which is the case if triggering was done by head-update or pullrequest), this expression will evaluate to the empty string otherwise, leaving the shell-script with a syntax-error:
```
if ;then
...
```